CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [12.0.1] - 2023-03-01
+## Changes
+- Fix get consent failures when bank send Revoked status enum.
```

Comment:
Can you please write how this will be achieved? It doesn't describe the problem or its solution.
It can be something like : "OB VRP v3.1.9 specification doesn't specify Revoked state for OBDomesticVRPConsentResponseData.StatusEnum. To support cases when ASPSP still send it adding VrpConsentResponseStatusEnumDeserializer to replace Revoked with Rejected state"

Reply:
No, it should be short but so that you don't need to do and read the code.
What is unclear in current comment:
Fix get consent failures(which consent? VRP or PISP?) when bank(what is a bank? we don't use bank term in ob-client) send Revoked status enum(which status enum? We have multiple of them).

Reply:
I tried to incorporate above change write changelog with bug description and your suggestion in [7a5751f](https://github.com/transferwise/openbanking-client/pull/180/commits/7a5751fbfdc46c88d1889807a5865b4b691270a4).